tests/scss/others/functional_argument_on_same_lines_in_maps.scss

```diff
@@ -0,0 +1,9 @@
+$display-breakpoints: map-deep-merge(
+  (
+    "print-only": "only print",
+    "screen-only": "only screen",
+    "xs-only": "only screen and (max-width: #{map-get($grid-breakpoints, "sm")-1})",
```

Comment:
I think it is some ugly, more readable (with spaces):
```
#{map-get($grid-breakpoints, "sm") - 1}
```

Reply:
Sure will make the changes